pkg/router/template/plugin.go

```diff
@@ -99,7 +99,7 @@ func NewTemplatePlugin(cfg TemplatePluginConfig) (*TemplatePlugin, error) {
 	globalFuncs := template.FuncMap{
 		"endpointsForAlias": endpointsForAlias,
 		"env":               env,
-		"matchString":       matchString,
+		"matchPattern":      matchPattern,
```

Comment:
Are there any docs for this that should change?  I couldn't find any.  So at the least can we put a comment over in the original PR that added matchString to say that it changed (and what the new behavior is).